code/game/atoms_movable.dm

```diff
@@ -1041,7 +1041,43 @@
 			if(. <= GRAB_AGGRESSIVE)
 				ADD_TRAIT(pulling, TRAIT_FLOORED, CHOKEHOLD_TRAIT)
 
+/**
+ * Adds the deadchat_plays component to this atom with simple movement commands.
+ *
+ * Returns the component added.
+ * Arguments:
+ * * mode - Either ANARCHY_MODE or DEMOCRACY_MODE passed to the deadchat_control component. See [/datum/component/deadchat_control] for more info.
+ * * cooldown - The cooldown between command inputs passed to the deadchat_control component. See [/datum/component/deadchat_control] for more info.
+ */
+/atom/movable/proc/deadchat_plays(mode = ANARCHY_MODE, cooldown = 12 SECONDS)
+	return AddComponent(/datum/component/deadchat_control/cardinal_movement, mode, list(), cooldown)
+
+/atom/movable/vv_get_dropdown()
+	. = ..()
+	VV_DROPDOWN_OPTION(VV_HK_DEADCHAT_PLAYS, "Start/Stop Deadchat Plays")
+
+/atom/movable/vv_do_topic(list/href_list)
+	. = ..()
+
+	if(!.)
```

Comment:
From quick glance this isn't how return value was used in this proc before ?

Reply:
Feast your eyes on this monstrosity

```
/obj/vv_do_topic(list/href_list)
	if(!(. = ..()))
		return
```

I think this is code I borrowed from Floyd. I can change it to be similar to the other vv_do_topic overrides

Reply:
I saw it in one place and there's a reason for it but I cant remember why.

Reply:
Oh wait I remember, if you don't do this double popups can occur, If the signal sends a response aka COMPONENT_VV_HANDLED, the parent returns TRUE, if this is the case, we shouldn't do anything else, else you get the case where clicking on the add_deadchat will popup twice.

Reply:
Not a fan how this is inconsistent between vv_do_topic implementations, but i guess that's for another PR